### ISSUE TITLE
StreamSets do not work locally

### ISSUE THREAD

**Issue by yorek** [Azure/usql#36]
It seems that stream sets don't work when submitting the job to the local machine. For example the 

[usql/Examples/AmbulanceDemos/AmbulanceDemos/5-Ambulance-StreamSets-PartitionedTables/5.1-Ambulance-StreamSets.usql](https://github.com/Azure/usql/blob/master/Examples/AmbulanceDemos/AmbulanceDemos/5-Ambulance-StreamSets-PartitionedTables/5.1-Ambulance-StreamSets.usql)

doesn't generate any output.
Is this correct? Or I'm missing something?
Thanks!

**Comment by MikeRys** [comment]
It should work now. What is the version you are using? Also, do you have the data copied into the right location on your local data root directory?

**Comment by yorek** [comment]
I'm using Azure Data Lake Tools for Visual Studio 2.0.7000.0. My root data directory is `D:\USQLDataRoot` and I haven't copied the files there. I've just executed the script from Visual Studio, just like as I did when I was just pointing to a single file.
Using the sample "Ambulance-1-1-QueryDriversOnADL.usql" as a starting poing, this works perfectly

`DECLARE @INPUT_DRIVERS string = "/Samples/Data/AmbulanceData/Drivers.txt";`

but this not

`DECLARE @INPUT_DRIVERS string = "/Samples/Data/AmbulanceData/{filename:*}.txt";`

since it creates and empty file.
Even copying the Ambulance data in the root data folder: `D:\USQLDataRoot\Samples\Data\AmbulanceData` it always generates an empty result.

**Comment by yorek** [comment]
I've managed to have it working. I took a look at the `ScopeCompilerTrace.log` and I've noticed that the `GetFolders `method looks in the local U-SQL data root. Moving the data manually did the trick. Yesterday I thought I did the same, but probably I've misspelled the path or something similar. Today I just copied and pasted the folder from the log and it works. 

If this is the way it is supposed to work, it would make sense to explicitly mention that files must be manually copied in the root folder in the help section on the top.
